lib/Dialect/FIRRTL/Transforms/LowerTypes.cpp

```diff
@@ -131,6 +136,10 @@ struct FIRRTLTypesLowering : public LowerFIRRTLTypesBase<FIRRTLTypesLowering>,
 
   // State to keep a mapping from (Value, Identifier) pairs to flattened values.
   DenseMap<ValueIdentifier, Value> loweredBundleValues;
+
+  // State to track the new attributes for the module.
+  SmallVector<NamedAttribute, 8> newAttrs;
```

Comment:
would it make sense to rename this to `newModuleAttrs` and `originalNumModuleArgs` for clarity?